tools/topology/topology2/include/pipelines/cavs/src-gain-playback.conf

```diff
@@ -0,0 +1,186 @@
+#
+# Src gain (volume) playback pipeline
+#
+# All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.src-gain-playback.N.attribute_name"
+#
+# Usage: src-gain-playback pipeline object can be instantiated as:
+#
+# Object.Pipeline.src-gain-playback."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/gain.conf>
+<include/components/cavs/src.conf>
+<include/components/pipeline.conf>
+
+Class.Pipeline."src-gain-playback" {
```

Comment:
kind reminder that we want all host pipelines to go into a mixer, and a different pipeline for dais.

I don't want yet another case where the mixer is ignored. 

To be clearer, multiple playback streams is a requirement.

Reply:
sure, I am working on it